### PR TITLE
Build & release arm64

### PR DESCRIPTION
**Pull request by carlcsaposs-canonical**

### CODE REVIEW

**Review by taurus-forever — APPROVED** [review]
Update README (same as snap) and please create backlog ticket to add tests here.

File: rockcraft.yaml [hunk 1]
```diff
@@ -32,30 +33,6 @@ parts:
...
-            ln -sf $LIB_DIR/libevent_pthreads-2.1.so.7 \
-                /usr/lib/x86_64-linux-gnu/libevent_pthreads-2.1.so.7
-            ln -sf $LIB_DIR/libprotobuf-lite.so.23 \
-                /usr/lib/x86_64-linux-gnu/libprotobuf-lite.so.23
-            ln -sf $LIB_DIR/libicudata.so.70 \
-                /usr/lib/x86_64-linux-gnu/libicudata.so.70
-            rm -rf /var/lib/mysql/
-            mysqld --initialize
-            cp -r /var/lib/mysql $CRAFT_STAGE/var/lib
-            chown 584788:584788 $CRAFT_STAGE/var/lib/mysql
```
Comment (carlcsaposs-canonical) [inline rockcraft.yaml:58]: I inspected the rock we're currently building (i.e. ghcr.io/canonical/charmed-mysql:8.0.37-22.04_edge), and it looks like it doesn't contain `/var/lib/mysql`
```
root@foo-0:/# find / -name mysql
/home/mysql
/usr/lib/x86_64-linux-gnu/perl5/5.34/DBD/mysql
/usr/lib/x86_64-linux-gnu/perl5/5.34/auto/DBD/mysql
/usr/lib/mysql
/usr/share/mysql
/usr/bin/mysql
/var/log/mysql
/etc/init.d/mysql
/etc/mysql
root@foo-0:/# mysqld --initialize-insecure -u mysql
root@foo-0:/# find / -name mysql
/home/mysql
/usr/lib/x86_64-linux-gnu/perl5/5.34/DBD/mysql
/usr/lib/x86_64-linux-gnu/perl5/5.34/auto/DBD/mysql
/usr/lib/mysql
/usr/share/mysql
/usr/bin/mysql
/var/log/mysql
/var/lib/mysql
/var/lib/mysql/mysql
/etc/init.d/mysql
/etc/mysql
```

so I think the entire `override-stage` in our rock is doing nothing

**Comment by carlcsaposs-canonical** [comment]
> please create backlog ticket to add tests here.

opened #47

**Review by taurus-forever — APPROVED** [review]

File: .github/workflows/release.yaml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        shell: python
+        run: |
+          import pathlib
+          import subprocess
+
+          for rock_file in pathlib.Path(".").glob("*.rock"):
+              subprocess.run(
+                  ["syft", rock_file.name, "--output", f"spdx-json={rock_file.name}.spdx.json"],
+                  check=True,
+              )
```
Comment (paulomach) [inline .github/workflows/release.yaml:56]: good ol shell?

```suggestion
        shell: bash
        run: |
          for rock_file in *.rock
          do
            syft "${rock_file}" --output spdx-json="${rock_file}".spdx.json
          done    
```

**Review by paulomach — APPROVED** [review]
good cleanup

File: .github/workflows/release.yaml (hunk 2 above)
Comment (carlcsaposs-canonical) [reply .github/workflows/release.yaml:56]: it'll probably move to reusable workflows at some point

to keep consistent with postgres repo, not changing for now

**Pull request merged**